ramls/item.json

```diff
@@ -377,7 +377,8 @@
       "type": "array",
       "description": "List of statistical code IDs",
       "items": {
-        "type": "string"
+        "type": "string",
+        "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[1-5][a-fA-F0-9]{3}-[89abAB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$"
```

Comment:
This is a compatibility breaking change to the interfaces that use this schema. Please change their major versions accordingly and update the dependents.

Reply:
Changed.

The same new interface version is used for enabling optimistic locking and the dependents get updated for optimistic locking by https://issues.folio.org/browse/UXPROD-3089 and https://issues.folio.org/browse/FOLIO-3241 . Therefore no additional work is needed to update dependents regarding this change (MODINVSTOR-755: Reject non-UUID statistical code IDs).

Reply:
Ok, whilst I think it doesn't match how compatibility breaking changes are typically made, I understand that this is a pragmatic decision to avoid the impact of breaking compatibility twice for a change that is likely to have limited impact in practice.